buildSrc/src/main/kotlin/androidx/build/ftl/RunTestOnFTLTask.kt

```diff
@@ -45,6 +50,7 @@ import javax.inject.Inject
  * Due to the limitations of FTL, this task only support application instrumentation tests for now.
  */
 @Suppress("UnstableApiUsage") // for gradle property APIs
+@CacheableTask
```

Comment:
Does this work if we have no `@Input`s specified?

Reply:
we have inputs here though?

also, what i tried was:

```
./gradlew :room:integration-tests:room-testapp-noappcompat:debugAndroidTestOnFirebaseTestLab --info --build-cache
./gradlew clean
./gradlew :room:integration-tests:room-testapp-noappcompat:debugAndroidTestOnFirebaseTestLab --info --build-cache
```
check that the second command didn't actually run the FTL test, instead the info said "FROM CACHE".
a bunch of other tasks still did run though :'( but not calling firebase would be big save here.